middleware/fastcgi/fcgiclient.go

```diff
@@ -373,6 +374,14 @@ func (this *FCGIClient) Request(p map[string]string, req io.Reader) (resp *http.
 	}
 	resp.Header = http.Header(mimeHeader)
 
+	if resp.Header.Get("Status") != "" {
```

Comment:
Yeah, I tried those and they are empty. It makes sense since this a new response object that is created earlier and only has the headers set.